Source/GUI/MemViewer/MemViewer.cpp

```diff
@@ -6,24 +6,29 @@
 #include <iomanip>
 #include <sstream>
 
+#include <QApplication>
 #include <QMouseEvent>
 #include <QPainter>
 #include <QScrollBar>
 
 #include "../../Common/CommonUtils.h"
 #include "../../DolphinProcess/DolphinAccessor.h"
 
+#define VISIBLE_COLS 16 // Should be a multiple of 16, or the header doesn't make sense
+#define VISIBLE_ROWS 16
+#define NUM_BYTES (VISIBLE_COLS * VISIBLE_ROWS)
+
```

Comment:
The last commit says the defines were replaced, but I still see them here.

Reply:
Forgot to push the latest changes, should be good now.